static/app/views/eventsV2/table/cellAction.spec.jsx

```diff
@@ -140,13 +144,20 @@ describe('Discover -> CellAction', function () {
     const wrapper = makeWrapper(view, jest.fn(), 9, defaultData, organization);
     wrapper.find('Container').simulate('mouseEnter');
 
-    it('toggles the context popover on click', function () {
+    it('toggles the context popover on click', async () => {
+      const api = new MockApiClient();
+      jest.spyOn(api, 'requestPromise').mockResolvedValue({status: 'ignored'});
+      // @ts-ignore useApi is mocked
+      useApi.mockReturnValue(api);
```

Comment:
Hey, there seemed to be an issue with using MockApiClient.addMockResponse compatibility with this test suite. Also, since this will be the **ONLY** test-case for quick context in this file (the rest being a part of the quickContext.spec.tsx) one valid mocked response should do .